<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>DuplicateOpOptionsWidget</class>
 <widget class="QWidget" name="DuplicateOpOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>193</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item column="0" row="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="cbHealing">
         <property name="text">
          <string>Healing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbPerspective">
         <property name="toolTip">
          <string>To correct perspective, first create a perspective grid.</string>
         </property>
         <property name="text">
          <string>Correct the perspective</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSourcePoint">
         <property name="toolTip">
          <string>Move the clone origin with the brush. Uncheck to keep cloning from the selected point.</string>
         </property>
         <property name="text">
          <string>Source point move</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbResetSourcePoint">
         <property name="toolTip">
          <string>Reset the origin every time you make a new stroke.</string>
         </property>
         <property name="text">
          <string>Source point reset before a new stroke</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkCloneProjection">
         <property name="toolTip">
          <string>When checked, clone from all visible layers. Otherwise, clone from the active layer.</string>
         </property>
         <property name="text">
          <string>Clone From All Visible Layers</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Clone Brush:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Select the source point from the current layer with Ctrl-click. Use Ctrl+Alt-click to select a source from the previously picked layer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="0" row="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>